0-ad-history-mod-package/simulation/templates/structures/han/temple.xml

```diff
@@ -10,6 +10,13 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Sìmiào</SpecificName>
+    <History>
+        The Han Empire's vast lands were places of cultural diversity and home to many beliefs and religions. The most widespread ones were Taoism and Buddhism. However, the central imperial government followed a legalistic and confucianistic ideology. Legalism claims the necessity of law, justice, and control, while Confucianism is all about morale, about reigning gently and obeying resistancelessly:
+        The Han political system was very complex but clearly structured. There was a very strict hierarchy, and great respect was demanded for higher-ranking people. Han emperors were, unlike many other emperors in history, not worshipped as gods but seen as Mandates of Heaven representing the gods.
+        But not only the official administration held a strong hierarchy; following the Confucian model, the man would lead a family, women and kids had little freedom, were clearly subordinate to the man, and had to obey him.
```

Comment:
OK, this paragraph I have problems with. But I'll discuss more in the forums, but even so the wording is really negative.

```suggestion
        It wasn't only the official administration that held a strong hierarchy. Following the Confucian model, the man would be the family's leader who women and children had to obey.
```